.github/workflows/kind_integration.yml

```diff
@@ -65,6 +68,35 @@ jobs:
         name: image-archives
         path: /home/runner/archives
   # todo: Keep in sync with `release.yml`
+  cli_windows_static_integration_tests:
+    name: Integration tests (windows)
```

Comment:
Should we even call this an integration test?

Reply:
Given we're testing the final product, the integration test appellation doesn't clash with me :man_shrugging: